src/org/usfirst/frc/team694/robot/commands/AcquirerLoosenHoldCommand.java

```diff
@@ -0,0 +1,24 @@
+package org.usfirst.frc.team694.robot.commands;
+
+import org.usfirst.frc.team694.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.InstantCommand;
+
+/**
+ *
+ */
+public class AcquirerLoosenHoldCommand extends InstantCommand {
+
+    public AcquirerLoosenHoldCommand() {
+        super();
+        requires(Robot.acquirer);
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    }
+
+    // Called once when the command executes
+    protected void initialize() {
+        Robot.acquirer.releaseCube();
```

Comment:
While you're at it, I'd consider thinking about what comes to mind when you say "release". The acquirer can technically "release" the cube by spinning the wheels backwards to push the cube out. (Just a suggestion)